parachains/overview.md

```diff
@@ -124,35 +122,36 @@ graph LR
-        B1[Identity]~~~B2[Balances]~~~B3[Sudo]~~~B4[EVM]
-        C1[Timestamp]~~~C2[Assets]~~~C3[Contracts]~~~C4[and more...]
+        A1[Aura]~~~A2[Parachain<br>System]~~~A3[Transaction<br>Payment]~~~A4[Sudo]
+        B1[Identity]~~~B2[Balances]~~~B3[Assets]~~~B4[EVM]
+        C1[Timestamp]~~~C2[Staking]~~~C3[Contracts]~~~C4[and more...]
     end
     AP --> SP
 ```
 
-### Cumulus
+### Cumulus: Parachain-Specific Functionality
 
-Cumulus provides utilities and libraries to turn FRAME-based runtimes into runtimes that can be a parachain on Polkadot. Cumulus runtimes are still FRAME runtimes but contain the necessary functionality that allows that runtime to become a parachain on a relay chain.
+Cumulus is what transforms a Polkadot SDK-based runtime into a parachain-capable runtime. It provides the essential components for communicating with the relay chain, participating in Polkadot's consensus, and handling parachain-specific operations like block validation and collation.
 
-## Why Use Polkadot SDK?
+Key Cumulus components include:
 
-Using the Polkadot SDK, you can build application-specific blockchains without the complexity of building a blockchain from scratch or the limitations of building on a general-purpose blockchain. You can focus on crafting the business logic that makes your chain unique and innovative with the additional benefits of flexibility, upgradeability, open-source licensing, and cross-consensus interoperability.
+- **Parachain System Pallet**: Core parachain functionality and relay chain communication.
+- **Collator Consensus**: Block production logic for parachain collators.
+- **Relay Chain Interface**: APIs for interacting with the Polkadot relay chain.
+- **Validation Data**: Handling proof-of-validity data required by relay chain validators.
 
-## Create a Custom Blockchain Using the SDK
+## Getting Started with Parachain Development
 
-Before starting your blockchain development journey, you'll need to decide whether you want to build a standalone chain or a parachain that connects to the Polkadot network. Each path has its considerations and requirements. Once you've made this decision, follow these development stages:
+Building a parachain requires understanding the relationship between your chain and the Polkadot-SDK-based relay chain. Key considerations include:
 
-```mermaid
-graph LR
-    A[Install the Polkadot SDK] --> B[Build the Chain]
-    B --> C[Deploy the Chain]
-```
+- [Designing your runtime logic and choosing appropriate pallets](/parachains/customize-your-runtime){target=\_blank}
+- [Implementing XCM for cross-chain communication](/parachains/interoperability){target=\_blank}
+- [Setting up collator infrastructure for block production](/nodes-and-validators){target=\_blank}
+- [Deploying your parachain by obtaining coretime](/parachains/launch-a-parachain){target=\_blank}
+- [Upgrading your parachain's runtime](/parachains/runtime-maintenance/runtime-upgrades){target=\_blank}
```

Comment:
```suggestion
- [Designing your runtime logic and choosing appropriate pallets](/parachains/customize-your-runtime/){target=\_blank}
- [Implementing XCM for cross-chain communication](/parachains/interoperability/){target=\_blank}
- [Setting up collator infrastructure for block production](/nodes-and-validators/){target=\_blank}
- [Deploying your parachain by obtaining coretime](/parachains/launch-a-parachain/){target=\_blank}
- [Upgrading your parachain's runtime](/parachains/runtime-maintenance/runtime-upgrades/){target=\_blank}
```

Trailing slashes should be at the end of these links. We're working on a way to enforce this in PRs